nrpy/helpers/colored.py

```diff
@@ -0,0 +1,65 @@
+"""
+Placeholder.
+
+Author: Steven Brandt
+"""
+
+import sys
+from typing import Any, Dict
+from typing_extensions import Literal
+
+color_names = Literal["red", "green", "yellow", "blue", "magenta", "cyan"]
+
+
+def not_colored(arg: Any, c: color_names) -> str:  # pylint: disable=unused-argument
+    """
+    Provide stringified version of the argument with coloring disabled.
+
+    :param arg: the object to convert to a plain string
+    :param c: a valid color name
+
+    :return: a stringified version of arg
+    """
+    return repr(arg)
+
+
+colors: Dict[color_names, str] = {
+    "red": "\033[31m",
+    "green": "\033[32m",
+    "yellow": "\033[33m",
+    "blue": "\033[34m",
+    "magenta": "\033[35m",
+    "cyan": "\033[36m",
+}
+reset = "\033[0m"
+
+
+def colored(arg: Any, c: color_names) -> str:
```

Comment:
How about this?
def colored(arg: Any, c: color_names) -> str:
    """
    Return the stringified version of the argument with the specified color.

    :param arg: The object to convert to a string
    :param c: The name of the color to use
    :return: A stringified and colored version of `arg`
    :raises AssertionError: If `c` is not a string or not a valid color name
    """
    assert isinstance(c, str), "Color name must be a string"
    assert c in colors, f"Invalid color name: {c}"
    return f"{colors[c]}{arg}{reset}"